README.md

```diff
@@ -14,6 +14,11 @@ Previous versions:
 
 [SemVer](https://semver.org) versioning applies to OCM, and all versions are supported unless stated otherwise by an implementation.
 
+## Invitation flow at a glance
```

Comment:
I suggest to not include this in the README, there are quite a few flows and the Invite is not even the core one, therefore it does not make much sense to show it here.